qc2md.py

```diff
@@ -100,6 +100,18 @@ def parse_args() -> argparse.Namespace:
         default=True,
         help="Display a picker interface if there are multiple matching reference lines (default: %(default)s)",
     )
+    parser.add_argument(
+        "--nondialogue-categories",
+        nargs="*",  # 0 or more
+        default="Typeset Encode",
```

Comment:
Using a string here *technically* works, but it's more coincidence than anything else; as in python, `"Typeset" in "Typeset Encode"` is ``True``. Would heavily recommend list/tuple, especially as using `nargs` will always output lists.